### ISSUE TITLE
Felix 0.6.3 appears to be crashing 

### ISSUE THREAD

**Issue by dwchub** [laoo/Felix#127]
Hi! I attempted to test out the latest release of Felix and I grabbed release 0.6.3 with the icon. When I run it, I see a white window open up briefly and then closes. I tried both Felix.exe and Felix32.exe and both gave me the same behavior. I had to unblock Windows from disallowing it to run first. Older versions start up correctly.

Currently using Windows 11 23H2
Here's what is shown in the Event Viewer:

> Faulting application name: Felix.exe, version: 0.0.0.0, time stamp: 0x668ab304
Faulting module name: MSVCP140.dll, version: 14.29.30153.0, time stamp: 0x653aa208
Exception code: 0xc0000005
Fault offset: 0x0000000000013080
Faulting process id: 0x0x7894
Faulting application start time: 0x0x1DAEDBCC7D852EC
Faulting application path: C:\Users\me\Documents\emu\felix\Felix.exe
Faulting module path: C:\WINDOWS\SYSTEM32\MSVCP140.dll
Report Id: 3eb7dc54-c15c-4ae9-a367-4f6aca4d4e77
Faulting package full name: 
Faulting package-relative application ID:

**Comment by 42Bastian** [comment]
Did you try starting it w/o file? What if you start it with an .lnx?

**Comment by dwchub** [comment]
I started by double clicking on it w/o a file. I even tried specifying a file by either dragging and dropping a .lnx file to it and attempted from the command line as well.
I just now tried deleting %APPDATA%\..\local\Felix and I see the same error in the Event Viewer.

It's definitely something specific to this particular computer. I've another laptop where I just tried it and it started fine and I can load a .lnx file.

I'll take look at this MSVCP140.dll file and see if I can get to the bottom of it.

Thank you.
